templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+  <li><a href="#">Home page</a></li>
+  <li><a href="#">Manufacturers</a></li>
+  <li><a href="#">Cars</a></li>
+  <li><a href="#">Drivers</a></li>

```

Comment:
The links currently use `#` as the href value. Consider replacing these with actual URLs or Django template tags like `{% url 'name_of_the_view' %}` to make the links functional.